src/routes/jira/jira-get-connected-repos.ts

```diff
@@ -0,0 +1,139 @@
+import { NextFunction, Request, Response } from "express";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+import { Subscription, TaskStatus } from "~/src/models/subscription";
+
+interface Page {
+	pageNum: number;
+	isCurrentPage: boolean;
+}
+
+export const JiraGetConnectedRepos = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+
+	try {
+		const { jiraHost, nonce } = res.locals;
+		const subscriptionId = Number(req.params.subscriptionId) || Number(req.body.subscriptionId);
+		const page = Number(req.query.page || 1);
+
+		if (!jiraHost) {
```

Comment:
I reckon we could get away without it? As should rely on the middleware?

Reply:
goneskies!